cartography/intel/aws/ssm.py

```diff
@@ -65,6 +81,16 @@ def get_instance_patches(
     return instance_patches
 
 
+def transform_instance_patches(data_list: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
+    for p in data_list:
+        p["Id"] = f"{p['_instance_id']}-{p['Title']}"
```

Comment:
The title can have hyphen and spaces. I think we should use the kbid instead.
```suggestion
        p["Id"] = f"{p['_instance_id']}/{p['KBId']}"
```

Reply:
Non-blocker: we could move the somewhat out-of-band transformation on line 79 to a loop within `sync()`.

- patch["_instance_id"] = instance_id

to something like 
```
...
for instance_id in instance_ids:
  data = transform_instance_patches(instance_id, data)
  load_instance_patches(neo4j_session, data, region, current_aws_account_id, update_tag)
...
```

Reply:
The old way of forming an ID used this way. Let's keep it the same so that this is a refactor that doesn't change something important like an id.

Reply:
For the out-of-band thing, good idea, we can include that later on